aspnet/mvc/controllers/testing/sample/TestingControllersSample/src/TestingControllersSample/ClientModels/IdeaDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace TestingControllersSample.ClientModels
+{
+    public class IdeaDTO
```

Comment:
Rather than making these properties camel case, one could change the formatters to emit data in camel case format...(ex: json.net serializer supports this)